### ISSUE TITLE
Focus states for links styled as buttons (e.g. previous, next in guides)

### ISSUE THREAD

**Issue by keelanfh** [localgovdrupal/localgov_base#395]
In our accessibility audit, it was suggested that the focus state changing the background of this link to white here makes it blend in, rather than making it more visible.

I'm not sure that it's a breach of the standard, but we might be able to do a bit better here.

![image](https://user-images.githubusercontent.com/19519457/222895416-71b41129-fd5e-4f37-9813-1430687ba926.png)

We could follow gov.uk's approach, which would make this more consistent with our other link focus states, though you could argue that this also makes the "button" blend in as it reduces the contrast between the button colour and background:

![image](https://user-images.githubusercontent.com/19519457/222896227-a83e937b-cded-4b0e-9445-769fe3267baf.png)

Another option would be some kind of box around the "button", which should probably also have a combination of yellow and black to ensure sufficient contrast against the "button" and the background. GOV.UK use this in some places.

**Comment by markconroy** [comment]
@keelanfh I think this might be an issue with your theme rather than with LGD itself. Our default is to set the background to yellow with a thick border when in :focus state. See this screenshot for illustration:

![Screenshot 2023-03-16 at 10 31 15](https://user-images.githubusercontent.com/2183332/225590239-910b4ed7-d8c0-4982-94b9-b96c42c3b53c.jpg)

**Comment by keelanfh** [comment]
Aha, it was there before, honest 🙈 - my screenshot was from the demo site. Funnily enough, I think this was fixed by #392 - the extra selector you added to `a:focus` means that this now takes precedence over the `background-color` set there...
